src/configs/nuage/scripts/pps-simulator/simulate_pps_stats.py

```diff
@@ -723,15 +759,22 @@ def main():
         "perf_mons": perf_mons
     }
 
+
+    import timeit
+    s_time = timeit.default_timer()
+    es_chunk_size = config.get('default','es_chunk_size')
+
     flow_stats = SimulateFlowStats(flowData, simData)
-    sla_flows = flow_stats.generate_flow_stats(startTime, endTime)
+    sla_flows = flow_stats.generate_flow_stats(startTime, endTime,es_chunk_size)
 
     probe_stats = SimulateProbeStats(flowData)
-    probe_stats.generate_probe_stats(startTime, endTime)
+    probe_stats.generate_probe_stats(startTime, endTime,es_chunk_size)
 
     sla_stats = SimulateSLAStats(flowData)
-    sla_stats.generate_sla_stats(sla_flows)
+    sla_stats.generate_sla_stats(sla_flows,es_chunk_size)
 
+    e_time = timeit.default_timer()
+    print("\n-------------------------\n\nRunning Time = %s"%(e_time-s_time))
```

Comment:
Overall good work Bharat. Its a considerable improvement on speed. 
Please remove the extra prints from this file.

Reply:
Sure, would remove the extra prints. Thanks